style.css

```diff
@@ -1,129 +1,239 @@
+  width: 0%;
+  height: 2px;
+  background: #f44336;
+  display: block;
+  margin: auto;
+  transition: 0.5s;
+}
+
+.nav-links__link:hover::after {
+  width: 100%;
+}
+
+.hero-text {
+  font-family: 'Roboto', sans-serif;
+  width: 90%;
+  color: #fff;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+.hero-text__heading {
+  font-size: 60px;
+}
+
+.hero-text__copy {
+  font-size: 15px;
+  margin: 10px 0 40px;
+}
+
+.hero-btn {
+  display: inline-block;
+  color: #fff;
+  border: 1px solid #fff;
+  padding: 10px 30px;
+  background: transparent;
+  text-decoration: none;
+}
+
+.hero-btn:hover {
+  background: #f44336;
+  border: 1px solid #f44336;
+  transition: 1s;
+}
+
+.navbar .fa {
+  display: none;
+}
+
+@media (max-width: 700px) {
+  .hero-text__heading {
+    font-size: 20px;
+  }
+
+  .nav-links__item {
     display: block;
-    margin: auto;
+  }
+
+  .nav-links {
+    position: absolute;
+    background: #f44336;
+    height: 100vh;
+    width: 200px;
+    top: 0;
+    right: 0;
+    text-align: left;
+    z-index: 1;
     transition: 0.5s;
+    display: none;
+  }
 
+  .navbar .fa {
+    display: block;
+    color: #fff;
+    margin: 10px;
+    font-size: 22px;
+    cursor: pointer;
+  }
+
+  .nav-links__list {
+    padding: 30px;
+  }
 }
-.nav-links ul li:hover::after{
-    width:100%;
 
+/* ------------about us page----- */
+
+.aboutus-header {
+  height: 50vh;
+  width: 100%;
+  background-image: linear-gradient(rgba(4, 9, 30, 0.7), rgba(4, 9, 30, 0.7)), url("images/background.jpg");
+  background-position: center;
+  background-size: cover;
+  text-align: center;
+  color: #fff;
 }
 
-.text-box{
-    font-family: 'Roboto', sans-serif;    
-    width: 90%;
-    color: #fff;
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%,-50%);
-    text-align: center;
+.heading-primary {
+  font-size: 48px;
+}
+
+.content-container {
+  max-width: 1140px;
+  margin: 0 auto;
+}
+
+.aboutus-subheader {
+  margin-top: 16px;
+  font-size: 20px;
+  padding: 32px 16px 0;
+}
+
+.aboutus-subheading {
+  font-size: 40px;
 }
 
-.text-box h1{
-    font-size: 60px;
+.title {
+  margin-top: 8px;
+  padding: 8px 16px 0;
 }
-.text-box p{
-    font-size: 15px;
-    margin: 10px 0 40px;
+
+.title__copy {
+  font-size: 40px;
+  margin-top: 32px;
 }
-.hero-btn{
+
+.about-row {
+  display: flex;
+  margin-bottom: 50px;
+}
+
+.about-col {
+  flex-basis: 50%;
+  padding: 32px 16px 0;
+}
+
+.para-title {
+  font-size: 18px;
+  font-weight: 700;
+  margin-bottom: 8px;
+}
+
+.para-copy {
+  font-size: 14px;
+  font-weight: 300;
+  margin-bottom: 32px;
+  text-align: justify;
+}
+
+.about-image {
+  width: 100%;
+  object-fit: cover;
+}
+
+.image-wrapper {
+  order: 1;
+}
+
+.hero-btn--color-red {
+  border: 1px solid #f44336;
+  background: transparent;
+  color: #f44336;
+}
+
+.hero-btn--color-red:hover {
+  color: #fff;
+}
+
+@media (max-width: 700px) {
```

Comment:
it's better to group styles by media query, so this @media block should go on top or bottom of the css file